src/runtime-rs/crates/resource/src/rootfs/mod.rs

```diff
@@ -4,23 +4,29 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+mod nydus_rootfs;
 mod share_fs_rootfs;
 
-use std::{sync::Arc, vec::Vec};
-
+use agent::Storage;
 use anyhow::{anyhow, Context, Result};
 use async_trait::async_trait;
+use hypervisor::Hypervisor;
 use kata_types::mount::Mount;
+use std::{sync::Arc, vec::Vec};
 use tokio::sync::RwLock;
 
 use crate::share_fs::ShareFs;
 
-const ROOTFS: &str = "rootfs";
+use self::nydus_rootfs::NYDUS_ROOTFS_TYPE;
 
+const ROOTFS: &str = "rootfs";
+const HYBRID_ROOTFS_WRITABLE_LAYER_LOWER_DIR: &str = "rootfs_lower";
```

Comment:
It's a bit confusing to call a lower layer writable. Can you rename it to `HYBRID_ROOTFS_LOWER_DIR`?

Reply:
fixed